main/res/values/strings.xml

```diff
@@ -252,6 +253,8 @@
     <string name="info_stored_image">New image saved to:</string>
     <string name="info_storing_static_maps">Trying to store static maps</string>
     <string name="info_cache_saved">The cache has been stored locally</string>
+    <string name="info_calculator_state_saved">Current state of the calculator has been saved to waypoint.</string>
```

Comment:
> "The calculation result was invalid, no coordinates could be set but the current state of your calculation has been saved to the waypoint"

Too long for a toast. I don't like positive feedback (in terms of UI).

> "Always save when 'Done' is pressed"

👍, but please without a toast. Keep toast for things that aren't expected.